internal/testplanet/run.go

```diff
@@ -74,9 +74,6 @@ func Run(t *testing.T, config Config, test func(t *testing.T, ctx *testcontext.C
 
 			planet.Start(ctx)
 
-			// make sure nodes are refreshed in db
```

Comment:
Are we making sure to wait for nodes to contact the satellite here?

Reply:
we wait for them here https://github.com/storj/storj/blob/master/internal/testplanet/planet.go#L219